proto/osmosis/concentrated-liquidity/tickInfo.proto

```diff
@@ -26,4 +26,15 @@ message TickInfo {
     (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.DecCoins",
     (gogoproto.nullable) = false
   ];
+  repeated UptimeTracker uptime_trackers = 4 [
+    (gogoproto.moretags) = "yaml:\"uptime_growth_outside\"",
+    (gogoproto.nullable) = false
+  ];
+}
+
+message UptimeTracker {
```

Comment:
Note to reviewer: this was the cleanest approach I could find to support `[]DecCoins` in proto. Currently it only includes `UptimeGrowthOutside`, but in the future it might include more incentive-specific information or other useful fields.